NEWS.md

```diff
@@ -7,6 +7,7 @@
 * Upgrade folio-kafka-wrapper to 3.0.0 version ([MODINV-834](https://issues.folio.org/browse/MODINV-834))
 * Add deduplication mechanism for sharing Instance ([MODINV-866](https://issues.folio.org/browse/MODINV-866))
 * Allow to overlay source 'MARC' instances without related MARC record ([MODINV-846](https://issues.folio.org/browse/MODINV-847))
+* Inventory cannot process Holdings with virtual fields ([MODINV-941](https://issues.folio.org/browse/MODINV-941))
```

Comment:
this should go under new iteration. v20.1.0 has been released, new iteration v20.2.0-SNAPSHOT